### PR TITLE
Documentation fixes for DBFS API 

### PR DESCRIPTION
**Pull request by a0x8o**
## What changes are proposed in this pull request?
Updated reference to DBFS API documentation to https://docs.databricks.com/dev-tools/api/latest/dbfs.html

## How is this patch tested?
- [X] Unit tests
- [X] Integration tests
- [X] Manual tests

### CODE REVIEW

File: GIT-PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+git branch -d newfeature
+```
+ 
+**Additional Reading**
+* [Atlassian - Merging vs. Rebasing](https://www.atlassian.com/git/tutorials/merging-vs-rebasing)
+
+**Sources**
+* [GitHub - Fork a Repo](https://help.github.com/articles/fork-a-repo)
+* [GitHub - Syncing a Fork](https://help.github.com/articles/syncing-a-fork)
+* [GitHub - Checking Out a Pull Request](https://help.github.com/articles/checking-out-pull-requests-locally)
```
Comment (williambrandler) [inline GIT-PROCESS.md:158]: cool, thanks for this. Did you write it yourself or adapt it from another source?

File: docs/source/getting-started.rst [hunk 2]
```diff
@@ -98,7 +98,7 @@ Here is how to set it up on the Databricks web application,
 3. Sync the Glow notebooks via Repos
 
    #. Fork the `Glow github repo <https://github.com/projectglow/glow>`_.
-   #. Clone your fork to your Databricks workspace using Repos (step-by-step `guide <https://docs.databricks.com/repos/index.html#clone-a-remote-git-repository>`_).
+   #. Clone your fork to your Databricks workspace using Repos (step-by-step `guide <https://docs.databricks.com/repos/sync-remote-repo.html>`_).
```
Comment (williambrandler) [inline docs/source/getting-started.rst:101]: thanks for updating these

**Review by williambrandler — APPROVED** [review]
lgtm

File: GIT-PROCESS.md (hunk 1 above)
Comment (a0x8o) [reply GIT-PROCESS.md:158]: You an cite me as author

**Comment by williambrandler** [comment]
> @williambrandler it seems some disconnect with the tests for completing for this PR has left it in limbo. Can you take a look at this and see if you can have the process re-run or 'bump' the tests?

hi @a0x8o yeah I am confused, why aren't the circle-ci tests running? @dvcastillo would you know?

**Pull request merged**